packages/components/Button/src/useButton.ts

```diff
@@ -24,7 +24,7 @@ export const useButton = (props: ButtonProps): ButtonInfo => {
        */
       ...(isDisabled && { disabled: isDisabled }),
       accessible: true,
-      accessibilityRole: 'button',
+      accessibilityRole: accessibilityRole || 'button',
```

Comment:
Couldn't this also be achieved by doing a `...rest` spread at the bottom of this hook (after line 34)?

Reply:
`rest` is used by `usePressableState`, and the result is mixed in _before_ the overrides here, so it looks like these could be intentional. I do see that certain other properties (`enableFocusRing`, `iconPosition`) do apply the same pattern that I am proposing to apply here.